packages/remark-lint-list-item-spacing/index.js

```diff
@@ -7,11 +7,16 @@
  *   Warn when list looseness is incorrect, such as being tight
  *   when it should be loose, and vice versa.
  *
+ *   Options: `Object`.
```

Comment:
Move this to just above the definition of the actual options.
Could you also mention it’s optional? Like so: ``Options: optional `Object`.``